spec/controllers/portal_controller_spec.rb

```diff
@@ -507,8 +507,8 @@
           expect(response.status).to eq(200)
         end
 
-        it 'is "null"' do
-          expect(response.body).to eq('null')
+        it 'is blank' do
+          expect(response.body).to eq('')
```

Comment:
This is not testing anything. The response body is always blank because the templates are not rendered in the controller specs. Instead test that it's rendered the promo_card template.